torch/csrc/jit/symbolic_script.cpp

```diff
@@ -114,6 +357,14 @@ c10::optional<GradientPair> gradientInfoForSchema(
     return cache_it->second;
   } else {
     auto schema_str = canonicalSchemaString(schema);
+    // JIT doesn't support keyword only arguments.
```

Comment:
can you make a follow up issue to tracking adding support for `, *,` to the parser.